conf/machine/narwhal.conf

```diff
@@ -12,7 +12,8 @@ MACHINE_HAS_WLAN = "true"
 PREFERRED_PROVIDER_virtual/android-system-image = "android"
 PREFERRED_VERSION_android = "oreo"
 
-PREFERRED_PROVIDER_virtual/kernel = "linux-firefish"
+PREFERRED_PROVIDER_virtual/kernel = "linux-narwhal"
 PREFERRED_VERSION_linux = "3.18+oreo"
 
-IMAGE_INSTALL += "underclock udev-droid-system wifi-enabler bluebinder asteroid-hrm"
+IMAGE_INSTALL += "underclock udev-droid-system wifi-enabler bluebinder asteroid-compass"
```

Comment:
We don't need `wifi-enabled` and `bluebinder` at this point.
I would assume that your journal is now being spammed with `bluebinder` related messages?